components/list/filtered-documents/index.vue

```diff
@@ -55,29 +75,71 @@
                                 data           : { type: Array  },
                                 filters        : { type: Array  },
                                 filterTitle    : { type: String },
-                                filterProperty : { type: String, default: 'region' }
+                                filterProperty : { type: String, default: 'region' },
+                                filters2        : { type: Array  },
+                                filterTitle2    : { type: String },
+                                filterProperty2 : { type: String, default: 'region' }
```

Comment:
wht `filters2...` and what the use. very unclear to me
cannnot just use `filters:[]` and make `?filterTitle:[]`